### ISSUE TITLE
Inherited methods are not included

### ISSUE THREAD

**Issue by jmac105** [midknight41/examine-instance#24]
See [proof here](https://runkit.com/embed/1fymg02ewlfc)

When examine-instance is run against an instance of a class which extended another class the methods from the parent (super) class are not included

**Comment by singhs020** [comment]
what is the use case for this? If we go down this path we should label them in something like inherited as they are not first class citizens for the instance.
Also, what will happen in case of multi-level inheritance?

**Comment by jmac105** [comment]
```
class Animal {

  walk() {
  }

  run() {
  }
}

class Dog extends Animal {

  woof() {
  }
}

const buddy = new Dog();
```

To me if I ran examine-instance against `buddy` it should return public methods as `walk, run, woof`. 
As a consumer of an instance of `Dog` I don't really care so much what the internal structure of a `Dog` is. When I go to call `buddy.run()` I don't need to know that `run` is actually a method on the parent (`Animal`) class, as far as I'm concerned it's just a method on the `Dog` class. I would therefore say multi-level inheritance should work the same, anything that appears to the consumer as a public method should be returned as such by this package. 


Also I hope you're doing well @simranjeet2589 !! :) Good to hear from you mate

**Comment by singhs020** [comment]
Good to hear from you too @jmac105 :)

**Comment by midknight41** [comment]
Agreed it should work this way @jmac105. I think the creation of this module predated real JS classes and they were added after the fact.

I'll have a look on the train if I can. Alternatively, PRs always welcome.

**Comment by jmac105** [comment]
I've actually implemented a (slightly messy) workaround for this already so leave it with me... will submit a PR when I have some time.
